src/desktop/assets/main_layout.styl

```diff
@@ -8,6 +8,8 @@
 @require '../components/recently_viewed_artworks'
 @require '../components/inquiry_questionnaire/stylesheets'
 @require '../components/view_in_room'
+@require '../components/multi_page_modal'
```

Comment:
This also accidentally got removed because it was included via a different component that got removed.